File: 1560-number-of-students-doing-homework-at-a-given-time/number-of-students-doing-homework-at-a-given-time.py
class Solution:
    def busyStudent(self, s: List[int], e: List[int], q: int) -> int:
        length=len(s)
        c=0
        for i in range(0,length):
             if(q>=s[i] and q<=e[i]):
                 c+=1;
        return c

File: 1560-number-of-students-doing-homework-at-a-given-time/number-of-students-doing-homework-at-a-given-time.java
class Solution {
    public int busyStudent(int[] s, int[] e, int q) {
        int n=s.length;
        int c=0;
        for(int i=0;i<n;i++){
            if(q>=s[i] && q<=e[i] ){
                c++;
            }
        }
        return c;
    }
}